### PR TITLE
Initial work on new tutorial

### PR DESCRIPTION
**Pull request by kaedroho**
First drafts of first two sections and page structure for remaining sections

Preview: https://deploy-preview-342--wagtail-localize-org.netlify.app/tutorial/1-project-set-up/

### CODE REVIEW

File: docs/tutorial/2-configure-wagtail-localize.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+Find the ``INSTALLED_APPS`` setting, and insert ``'wagtail_localize'`` and ``'wagtail_localize.locales'`` in between ``'search`` and ``'wagtail.contrib.forms'``:
+
+``` python
+
+INSTALLED_APPS = [
+    'home',
+    'search',
+
+    # Insert these here
+    'wagtaiil_localize',
```
Comment (patillacode) [inline docs/tutorial/2-configure-wagtail-localize.md:20]: typo!
`wagtaiil_localize` 🔴  vs `wagtail_localize` 🟢

File: docs/tutorial/2-configure-wagtail-localize.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+![Wagtail explorer menu with internationalisation enabled](/_static/tutorial/wagtail-explorer-with-i18n.png)
+
+## Create the "French" locale
+
+Wagtail Localize links all content to ``Locale`` objects in the database. This allows the language codes to be updated
+easily and also allows custom components (such as currencies, flags, etc) to be associated with them in the database.
+
+Wagtail Localize sets up only one ``Locale`` to begin with, which is the one the corresponds to the ``LANGUAGE_CODE``
+setting. In this tutorial, we left ``LANGAUGE_CODE`` set to ``en-gb`` so the initial ``Locale`` would've been created
```
Comment (patillacode) [inline docs/tutorial/2-configure-wagtail-localize.md:165]: typo!
🔴  `LANGAUGE_CODE` vs `LANGUAGE_CODE ` 🟢

File: docs/tutorial/2-configure-wagtail-localize.md (hunk 1 above)
Comment (kaedroho) [reply docs/tutorial/2-configure-wagtail-localize.md]: Fixed, thanks!

File: docs/tutorial/2-configure-wagtail-localize.md (hunk 2 above)
Comment (kaedroho) [reply docs/tutorial/2-configure-wagtail-localize.md]: Fixed, thanks!

File: docs/tutorial/3-content.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+Navigate to the English Home page in the explorer, click "Add child page" then click "Blog index page". Fill it in like
+the following screenshot, and publish it.
+
+![Blog index page being edited in Wagtail](/_static/tutorial/wagtail-blog-index-page.png)
+
+Then navigate to the new blog index page, click "Add child page" then click "Blog post page". Fill it in how you like,
+but make sure that you cover the following:
+
+ - Upload an image and add it to the "Image" field
+ - Create a "Blog Category" and select it in hte "Category" field
```
Comment (KalobTaulien) [inline docs/tutorial/3-content.md:126]:
```suggestion
 - Create a "Blog Category" and select it in the "Category" field
```

**Review by KalobTaulien — APPROVED** [review]
One wee typo, non-blocking. Thank you for writing this whole tutorial!

**Pull request merged**